### ISSUE TITLE
rangechanged event fires repeatedly on zoom in timeline

### ISSUE THREAD

**Issue by JoakimSoderberg** [almende/vis#397]
First of all, awesome library!

When dragging the timeline, the `rangechange` event fires continuously and `rangechanged` fires only once when you stop.

However, when zooming `rangechanged` keeps firing continuously, not only when the zoom has finished. 

Is this intentional? Because it would be nice to have the same behaviour when zooming. I'm sending the time range shown in the timeline via websocket to the server on the `rangechanged` event which then returns all events in that range. I would prefer this happening once the zoom action has finished, instead of continuously.

P.S.
@lnilsen in #346 mentioned having an event that differentiates between `rangechange` events when zooming and dragging the timeline, maybe this could be a new type of event?
